core/internal/features/ocr2/features_ocr2_test.go

```diff
@@ -190,6 +190,7 @@ func setupNodeOCR2(
 
 func TestIntegration_OCR2(t *testing.T) {
 	t.Parallel()
+	testutils.SkipFlakey(t, "https://smartcontract-it.atlassian.net/browse/BCF-3417")
```

Comment:
This is the wrong test - that's why the merge queue failed
You want `TestIntegration_OCR2_plugins`